dist/performance.js

```diff
@@ -11,34 +23,105 @@ function getLcpElement() {
             resolve(naiveLcpEntry);
         }).observe({ type: "largest-contentful-paint", buffered: true });
     }).then(({ startTime, element, url, size, loadTime, renderTime }) => {
-        let attributes = [];
-        for (let index = 0; index < element.attributes.length; index++) {
-            const ele = element.attributes.item(index);
-            attributes[index] = { name: ele.name, value: ele.value };
-        }
-
+        const attributes = getAttributes(element);
+        const styles = getComputedStyles(element, ['background-image']);
```

Comment:
Are we likely to want any more than this? Or should we just called this background-image?

```suggestion
        const backgroundImage = getComputedStyles(element, ['background-image']);
```

Reply:
Yeah I think it's likely that this will be used more.